x-pack/test/security_solution_api_integration/package.json

```diff
@@ -7,22 +7,27 @@
   "scripts": {
     "exception_workflows:server:serverless": "node ../../../scripts/functional_tests_server.js --config ./test_suites/detections_response/default_license/exceptions/workflows/configs/serverless.config.ts",
     "exception_workflows:runner:serverless": "node ../../../scripts/functional_test_runner  --config=test_suites/detections_response/default_license/exceptions/workflows/configs/serverless.config.ts --grep @serverless --grep  @brokenInServerless --invert",
```

Comment:
Not necessarily for this PR, but this is going to get huge as we move over all tests, maybe we can generalize it to a script where you pass in the folder name, script type (server or runner) and environment.

Reply:
I see your concern!, good idea